co-authors-plus.php

```diff
@@ -1338,6 +1340,31 @@ public function is_valid_page() {
 		return (bool) in_array( $pagenow, $this->_pages_whitelist );
 	}
 
+	/**
+	 * Builds list of capabilities that CAP should filter.
+	 *
+	 * Will only work after $this->supported_post_types has been populated.
+	 * Will only run once per request, and then cache the result.
```

Comment:
I really the explanation here. I would add something like:
Cache the result in $to_be_filtered_caps since CoAuthors_Plus is only instantiated once and stored as a global
or something like that to make it even more clear why this is cached for the whole request.

Reply:
Yep - done! :)